atlp-rwanda/fan7-bn-backend
# #170764589 Assign roles to users

### What does this Pr do ?
Super administrator should be able to assign roles to users
### Description of the task to be completed ?
- check if it is a super administrator
- check if user exist
- assign the new role
- unit test our endpoint
- To make sure the right people consumes this API, who ever is hitting this endpoint should only use his/her  JWT token sent in headers that they received when they created credentials to this API or Sign in to it.
- Admin should receive an message specifying that the red-flags/interventions was successfully updated or an error with description if it didn't.

And also the data provided should meet this API validation expectations

have the following endpoint work 
- PATCH /api/auth/assignRole

### How should this be manually tested?
after cloning this repository, od into it and RUN `npm i` after `npm start`
- Using postman test the endpoint above

Set key: Content-type value: application/json
make sure you have a JWT token in the headers with key:token value: JWT token generated while signing up or signing in as a super administrator
### Any background context you want to provide?
N/A
### What are the relevant pivot tracker stories ?
#170764589
https://www.pivotaltracker.com/story/show/170764589

#### screenshots if available?
![image](https://user-images.githubusercontent.com/37307172/73433936-73f27100-434e-11ea-8efa-e4da404d8624.png)
![image](https://user-images.githubusercontent.com/37307172/73434488-83be8500-434f-11ea-936f-785f636fdffe.png)
